ts/routes/image-occlusion/tools/lib.ts

```diff
@@ -63,12 +63,19 @@ export const groupShapes = (canvas: fabric.Canvas): void => {
 
     const activeObject = canvas.getActiveObject() as fabric.ActiveSelection;
     const items = activeObject.getObjects();
+
+    // @ts-expect-error not defined
+    const minOrdinal = Math.min(...items.map((item) => item.ordinal));
```

Comment:
Something that might be nice in the future (not in this PR) is to tell TypeScript about the additional properties we've added to these objects, so we don't need to ignore errors/cast them to any each time we access them. I think it may be possible in a .d.ts file?

Reply:
This can be done, I will look into this.